src/AppBundle/Entity/DomainName.php

```diff
@@ -37,6 +37,13 @@ class DomainName
      */
     private $name;
 
+    /** @var string
+     *
+     * @ORM\Column(name="path", type="string", length=255, nullable=true)
+     * @Assert\Regex("/^[\w\d]+[-_%.\/\w\d]*$/")
+     */
+    private $path;
```

Comment:
I would recommend enforcing the leading slash `/` to be consistent with the strict definition of an URI path.
Might come in handy for future uses and may be useful to match root  path `/` in some cases ?